Provide complete configuration for IntelliJ IDEA

### Expected behavior
Compiling and running phoenicis following the instructions on https://phoenicisorg.github.io/phoenicis/Developers/intellij-idea

### Actual behavior
I was able to build the project with maven "Maven Projects Build phoenicis (root) → Lifecycle → package" (I guess). But failed at creating a working run configuration.

I tried to follow https://phoenicisorg.github.io/phoenicis/Developers/intellij-idea which mentions java 11 and VM options with `--add-modules=ALL-MODULE-PATH --module-path=phoenicis-dist/target/lib` and a second set of VM options which is the same but `--module-path phoenicis-dist/target/lib` without the `=` sign.

There is a broken image-link which once seemed to point to a screenshot with the configuration example. I've found a image on https://github.com/PhoenicisOrg/phoenicis/blob/master/docs/images/intellij-idea-run.png . The configuration in this image uses java 8 without VM options. Additionally there is this setting on "Working directory" which isn't mentioned in the documentation.

I've tried all kinds of combination of all of the above. I'm insure which Stacktraces to upload.
Java 8 doesn't like any combination of `--add-modules` or `--module-path` and refuses to start. Java 11 will throw a very long Exception which starts with:
```
Exception in Application start method
java.lang.RuntimeException: Exception in Application start method
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:900)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:195)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mainController' defined in org.phoenicis.javafx.controller.ControllerConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.phoenicis.javafx.controller.MainController]: Factory method 'mainController' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'libraryController' defined in org.phoenicis.javafx.controller.ControllerConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.phoenicis.javafx.controller.library.LibraryController]: Factory method 'libraryController' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'viewLibrary' defined in org.phoenicis.javafx.views.ViewsConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.phoenicis.javafx.views.mainwindow.library.LibraryView]: Factory method 'viewLibrary' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'scriptInterpreter' defined in org.phoenicis.scripts.ScriptsConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.phoenicis.scripts.interpreter.ScriptInterpreter]: Factory method 'scriptInterpreter' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'nashornInterpreter' defined in org.phoenicis.scripts.ScriptsConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.phoenicis.scripts.interpreter.ScriptInterpreter]: Factory method 'nashornInterpreter' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'scriptEngineFactory' defined in org.phoenicis.scripts.ScriptsConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.phoenicis.scripts.nashorn.NashornEngineFactory]: Factory method 'scriptEngineFactory' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'setupWizardFactory' defined in org.phoenicis.scripts.wizard.WizardConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.phoenicis.scripts.wizard.UiSetupWizardFactory]: Factory method 'setupWizardFactory' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'setupUiFactory' defined in org.phoenicis.javafx.views.scriptui.JavaFxUiConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.phoenicis.scripts.ui.SetupUiFactory]: Factory method 'setupUiFactory' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'viewInstallations' defined in org.phoenicis.javafx.views.ViewsConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.phoenicis.javafx.views.mainwindow.installations.InstallationsView]: Factory method 'viewInstallations' threw exception; nested exception is java.lang.IllegalAccessError: class javafx.scene.control.Tab (in unnamed module @0x68d94ce6) cannot access class com.sun.javafx.event.EventHandlerManager (in module javafx.base) because module javafx.base does not export com.sun.javafx.event to unnamed module @0x68d94ce6
...
``` 
which I'm unsure if this is due to the start options.

After 1.5 hour gave up in frustration :P 

### System information
* I'm using git phoenicis-master
* Ubuntu 18.04
* 11.0.1 OpenJDK Runtime Environment 18.9 (build 11.0.1+13)
* Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
* IntelliJ IDEA 2018.3.4 CE Build #IC-183.5429.30 built on January 29, 2019

--- a/docs/_docs/Developers/intellij-idea.md
+++ b/docs/_docs/Developers/intellij-idea.md
@@ -15,7 +15,8 @@ Build phoenicis (root) → Lifecycle → package
 ### Run
 Run → Edit Configurations
 add: JavaFXApplication with:
-* main class: org.phoenicis.javafx.JavaFXApplication
+* Name: `JavaFXApplication`
+* Main class: `org.phoenicis.javafx.JavaFXApplication`
 * VM options: 
 ```
 --add-modules=ALL-MODULE-PATH --module-path=phoenicis-dist/target/lib
@@ -24,7 +25,15 @@ add: JavaFXApplication with:
 ```
 --add-modules=ALL-MODULE-PATH --module-path phoenicis-dist/target/lib
 ```
-![IntelliJ IDEA configuration](/images/intellij-idea-run.png)
+* Program arguments: empty
+* Working directory: `/path/to/phoenicis`
+* Environment variables: empty
+* Redirect input from: unchecked
+* Use classpath of module: `phoenicis-dist`
+* Include dependencis with "Provided" scope: unchecked
+* JRE: `Default`
+* Shorten command line: `user-local: none - java [options]`
+* Enable capturing form snapshots: unchecked
 
 ### Code Style
 File → Settings → Editor → Code Style → Scheme: Manage... → Import → Eclipse XML Profile: select [settings/POL_Formatter_Settings.xml](https://github.com/PhoenicisOrg/phoenicis/blob/master/settings/POL_Formatter_Settings.xml)

